BardMusicPlayer.Ui/ViewModels/MainViewModel.cs

```diff
@@ -3,44 +3,35 @@
  * Licensed under the GPL v3 license. See https://github.com/BardMusicPlayer/BardMusicPlayer/blob/develop/LICENSE for full license information.
  */
 
-using System;
-using System.ComponentModel;
-using System.Windows;
 using BardMusicPlayer.Coffer;
 using BardMusicPlayer.Grunt;
 using BardMusicPlayer.Pigeonhole;
 using BardMusicPlayer.Seer;
-using BardMusicPlayer.Ui.Controls;
 using BardMusicPlayer.Ui.Utilities;
+using BardMusicPlayer.Ui.Views;
+using Stylet;
 
-namespace BardMusicPlayer.Ui
+namespace BardMusicPlayer.Ui.ViewModels
 {
-    /// <summary>
-    /// Interaction logic for MainView.xaml
-    /// </summary>
-    public partial class MainView : Window
+    public class MainViewModel : Screen
     {
-        public MainView()
+        protected override void OnViewLoaded()
         {
-            InitializeComponent();
-            
-            BmpPigeonhole.Initialize(@Globals.DataPath + @"\Configuration.json");
+            var view = (MainView) View;
+            BmpPigeonhole.Initialize(Globals.DataPath + @"\Configuration.json");
 
-            LogManager.Initialize(new LogTextWriter(Log));
+            LogManager.Initialize(new(view.Log));
```

Comment:
There needs to be a better way to handle this, instead inherit the `TextBox` class and use the included Event Aggregator and make it `INotificationHandler<Log>` some sort of log instead.

Reply:
that's fine with me if you want to change the way the logging system will work as long as it's 100% non blocking async for backend services reporting logs